### ISSUE TITLE
Moving Character Saves

### ISSUE THREAD

**Issue by Noleafcloverx** [diasurgical/devilutionX#960]
**Describe**

Can I not move my characters from one computer to another?

**To Reproduce**
2. Click on '...'
3. Scroll down to '...'

**Expected behavior**

**Screenshots**

**Additional context**

**Comment by qndel** [comment]
you can ...

**Comment by Noleafcloverx** [comment]
It isn't working though.  I just did it and moved them to where you say to move them and they don't show up in game...

**Comment by qndel** [comment]
and where would that be?

**Comment by Noleafcloverx** [comment]
...in your readme....

# Save Games and configurations
The configurations and save games are located in:
C:\Users\[username]\AppData\Roaming\diasurgical\devilution

**Comment by qndel** [comment]
dude, you totally ignored the issue template, there's 0 guarantee you didn't put the saves on C:/shrekbutthole ... I know what's in our readme ...

**Comment by Noleafcloverx** [comment]
Okay...you can chill...I am not saying you are a terrible person...just asking for help is all...wow

**Comment by qndel** [comment]
and now I'll have to ask 10 questions because you didn't give any info 🙄 
are you on windows? what devilutionx version?

**Comment by qndel** [comment]
okay, you aren't going to get a next reply from me, follow the issue template next time 👍 good luck

**Comment by Noleafcloverx** [comment]
Wow...what is your problem?  I'm sorry you are having a bad day.  I'll figure it out don't worry.

**Comment by qndel** [comment]
**Expected behavior**
Problem solved.

**Comment by Noleafcloverx** [comment]
Or could it be that your issue template is completely ludicrously tedious?  This I am sure, was a very simple question to answer...not sure what has your panties in a bunch but good luck with life.

**Comment by qndel** [comment]
Sorry for being rude earlier, you seem to be a busy man, appreciate you found the time to open an issue, you don't have to provide any info, I'm getting so much $ for devilutionx that I'm going to test all possible versions and settings to reproduce your issue despite the fact that it's new year, to better suit the needs of busy people like you, the next issue template will be just a question "Does it work?" and two big green "yes"/"no" buttons :)

**Comment by AJenbo** [comment]
The sections are optional, but lacing them with the template text makes it even less easy to see what your issue is.

If it doesn't work for you the please provide the os and version of the two systems as well as the actual system paths you are using.

The easiest way is to check that it's they same folder as where the game generated the diablo.ini file

**Comment by Trihedraf** [comment]
The easiest way I have found to deal with portable saves is to have a save folder with your devilutionx.exe and make .bat file to launch with the --save-config option.

If you are on the 1.1.0 release this will save the saves and config to the folder "save"

     "devilutionx.exe" --save-dir save

If you are on the latest master saves and config are splitup

     "devilutionx.exe" --save-dir save --config-dir save

**Comment by qndel** [comment]
don't use bat files in 2021 for that 🙄 you can create a shortcut and pass the arguments there

**Comment by Trihedraf** [comment]
I personally hate shortcuts. They have be to a hard path, whereas a bat file can have just the program name and be moved with the exe anywhere.

**Comment by Noleafcloverx** [comment]
Thank you for the responses, I appreciate everyone's patience with someone who is new to this forum.  I did manage to figure it out on my own, for some reason the game had saved all blank character files and when I moved them they remained blank.  I was able to restore them and move them as normal.  I am unsure about what caused the game to save blank files.  I am running Windows 10 and using version 1.1.0.
